test/test-search.ts

```diff
@@ -0,0 +1,105 @@
+import test from 'ava';
+import assert from 'assert';
+import * as search from '../src/search';
+
+test(
+    'searchValue should have http module in standard require' +
+        'http case',
+    async t => {
+      const content = 'const a = require(\'http\');';
+      const result = await search.search(content);
+      t.true(result.requiredModules.has('http'));
+    });
+    
+test(
+    'searchValue should not have http module when there are no require calls',
+    async t => {
+      const content = 'console.log(\'http\')';
+      const result = await search.search(content);
+      t.true(result.requiredModules.size === 0);
+    });
+
+test(
+    'searchValue should have http module when require arg in template literals',
+    async t => {
+      const content1 = 'const a = require(`${\'http\'}`);';
+      const result1 = await search.search(content1);
+      t.true(result1.requiredModules.has('http'));
+
+      const content2 = 'const a = require(`http`);';
+      const result2 = await search.search(content2);
+      t.true(result2.requiredModules.has('http'));
+    });
+
+test('searchValue should have http and fs module', async t => {
+  const content =
+      'const a = require(\'http\');\n const b = require(`${\'fs\'}`);';
+  const result = await search.search(content);
+  t.true(result.requiredModules.has('http'));
+  t.true(result.requiredModules.has('fs'));
+});
+
+test('line number is accurate for search', async t => {
+  const content =
+      'const a = require(\'http\');\n const b = require(`${\'fs\'}`);';
+  const result = await search.search(content);
+
+  const httpPosition = (result.requiredModules.get('http'));
+  
+  if(httpPosition !== undefined && assert(httpPosition !== null, 'position of http is null') ){
```

Comment:
Use `t.true` instead of `assert`?

Reply:
Also, does `assert` (or `t.true`) return a value? I think they need to be on a separate line rather than in a conditional statement.